source/Irrlicht/CQuantNormalCache.cpp

```diff
@@ -0,0 +1,181 @@
+#include "CQuantNormalCache.h"
+
+namespace irr
+{
+namespace asset
+{
+
+core::vectorSIMDf CQuantNormalCache::findBestFit(const uint32_t& bits, const core::vectorSIMDf& normal) const
+{
+	core::vectorSIMDf fittingVector = normal;
+	fittingVector.makeSafe3D();
+	fittingVector = abs(fittingVector);
```

Comment:
there should be a requirement that the input to this function already be a an absolute value vector

Reply:
because `bestFit` returned will be an abs value vector